py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/configuration.py

```diff
@@ -13,71 +19,32 @@ class Configuration(Enum):
     https_mbta_busloc_s3.s3.amazonaws.com_prod_VehiclePositions_enhanced.json.gz
     https_mbta_integration.mybluemix.net_vehicleCount.gz
     """
-    RT_ALERTS = auto()
-    RT_TRIP_UPDATES = auto()
-    RT_VEHICLE_POSITIONS = auto()
-
-    @classmethod
-    def from_filename(cls, filename: str):
-        if 'mbta.com_realtime_Alerts_enhanced' in filename:
-            return cls.RT_ALERTS
-        if 'mbta.com_realtime_TripUpdates_enhanced' in filename:
-            return cls.RT_TRIP_UPDATES
-        if 'mbta.com_realtime_VehiclePositions_enhanced' in filename:
-            return cls.RT_VEHICLE_POSITIONS
-
-        raise Exception("Bad Configuration from filename %s" % filename)
-
-    def get_schema(self) -> pyarrow.schema:
-        if self == self.__class__.RT_VEHICLE_POSITIONS:
-            return pyarrow.schema([
-                ('year', pyarrow.int16()),
-                ('month', pyarrow.int8()),
-                ('day', pyarrow.int8()),
-                ('hour', pyarrow.int8()),
-                ('feed_timestamp', pyarrow.int64()),
-                ('vehicle_timestamp', pyarrow.int64()),
-                ('vehicle_id', pyarrow.string()),
-                ('vehicle_label', pyarrow.string()),
-                ('current_status', pyarrow.string()),
-                ('current_stop_sequence', pyarrow.int16()),
-                ('stop_id', pyarrow.string()),
-                ('position', pyarrow.struct([
-                    ('latitude', pyarrow.float64()),
-                    ('longitude', pyarrow.float64()),
-                    ('bearing', pyarrow.float32()),])),
-                ('trip', pyarrow.struct([
-                    ('trip_id', pyarrow.string()),
-                    ('route_id', pyarrow.string()),
-                    ('direction_id', pyarrow.int8()),
-                    ('schedule_relationship', pyarrow.string()),
-                    ('start_date', pyarrow.string()),
-                    ('start_time', pyarrow.string())])),
-                ('consist_labels', pyarrow.list_(pyarrow.string()))
-            ])
-
-        raise Exception("No Current Schema for %s" % self.name)
+    def __init__(self, filename: str) -> None:
+        self.file_path = Path(filename)
```

Comment:
i don't think we want a `Confiiguration` object to hold onto the path. i imagine we'll use the came `Configuration` to process a number of input files.

i think its fine to initialize with a filepath. we could also have a static method that derives the `ConfigType` from a filepath and we use that to initialize the `Configuration`.

Reply:
Agree, no reason to save the file path right now.

We haven't really talked about how we'll implement the processing of multiple files, but we'll need some kind of container object which I imagine will be responsible for keeping a list of files that were downloaded/processed and concatenating all of the data extracted from the gzip files into one object.